src/lib/helpers.js

```diff
@@ -12,3 +14,7 @@ export const getAccountLogin = account => {
     return account.auth.login || account.auth.identifier || account.auth.email
   }
 }
+
+export const redirectToStore = () => {
+  cozy.client.intents.redirect('io.cozy.konnectors', { type: 'konnector' })
```

Comment:
It feel weird to pass `io.cozy.konnectors` and `{ type: konnector }`. It feels like saying: 
> Yeah give me all konnectors of type konnectors please

It feels like we are accepting destination app constraints, not we are asking the stack to handle something

Instead, should it be possible that the Store provide an url `/#/konnectors/` (I think I know the response and it's something like "Yes it's possible but it is too much work")

Reply:
So now the store only handle the doctype `io.cozy.apps`, since we want the webapps and the konnectors to be considered as `applications` but others.